pkg/pubsub/pubsub.go

```diff
@@ -38,10 +37,11 @@ func (c *CrawlerPubSub) StopPublishers() error {
 }
 
 // BuildMessage builds a message for the publisher
-func (c *CrawlerPubSub) BuildMessage(timestamp int64, hash string) (*cpubsub.GooglePubSubMsg, error) {
+func (c *CrawlerPubSub) BuildMessage(newsroomException bool,
+	contractAddress string) (*cpubsub.GooglePubSubMsg, error) {
 	msg := CrawlerPubSubMessage{
-		Timestamp: timestamp,
-		Hash:      hash,
+		NewsroomException: newsroomException,
```

Comment:
We can just leave this for now, but do you think it would be useful to add a field for `msgType` that would have values like `newsroom-exc` and `run-proc` as const strings?   I'm just thinking if we need to send different types of messages down this same topic in the future to trigger other functions and it might be easier to use an explicit field than rely on absence of values.